ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.9.yaml

```diff
@@ -66,7 +66,8 @@ tests:
           cpu: "2"
           memory: 600Mi
     workflow: ipi-aws
-- as: sanity
+- api: true
```

Comment:
@jianzhangbjz I know #38247 is `- api: true`, but I would like to try if 
```yaml
- as: sanity
  api: true
```
also works.

So, could you please change it to the above and keep gcp-upi-xpn-p3-f360-destructive as 
```yaml
- api: true
  as: gcp-upi-xpn-p3-f360-destructive
```
?

if 
```yaml
- as: sanity
  api: true
```
works, I would like to use this format for our test job late because it possibly keep compatible tool parse.

Reply:
This format will be changed to the below automatically after running `make jobs  && make ci-operator-config`.
```yaml
- api: true
  as: sanity
```